packages/core/src/components/Icon/Icon.tsx

```diff
@@ -3,11 +3,11 @@ import cx from "classnames";
 import React, { CSSProperties, forwardRef, Ref } from "react";
 import useMergeRef from "../../hooks/useMergeRef";
 import { IconType as IconTypeEnum } from "./IconConstants";
-import { IconType } from "./Icon.types";
+import { IconType, MouseEventCallBack, SubIcon } from "./Icon.types";
```

Comment:
also sound like something very generic that should be in types rather in Icon.types (MouseEventCallBack)